axelor-marketing/src/main/resources/views/TargetList.xml

```diff
@@ -29,8 +29,14 @@
 		  	 	<editor x-show-titles="false">
 	            <field name="metaField" onSelect="action-target-list-partner-filters-field-domain" colSpan="3" widget="SuggestBox" required="true" onChange="action-filter-method-update-target-field" target-name="label"/>
 	            <field name="targetField" colSpan="3" showIf="metaField.relationship != null" requiredIf="metaField.relationship != null" onChange="action-filter-method-update-target-type"/>
-	            <field name="operator"  colSpan="3" required="true"/>
-	            <field name="value" colSpan="3" onChange="action-studio-filter-validate-values"/>
+	            <panel name="partnerFilterListOperatorPanel" colSpan="3">
+	            	<field name="operator" colSpan="12" required="true" selection-in="['=','!=','isNull','notNull']" hidden="true" showIf="targetType == 'String'"/>
+		            <field name="operator" colSpan="12" required="true" selection-in="['=','!=','&gt;=','&lt;=','&gt;','&lt;','isNull','notNull']" hidden="true" showIf="targetType == 'Integer' || targetType == 'Long' || targetType == 'BigDecimal' || targetType == 'LocalDateTime' || targetType == 'LocalDate'"/>
+		            <field name="operator" colSpan="12" required="true" selection-in="['isTrue','isFalse']" hidden="true" showIf="targetType == 'Boolean' || targetType == 'byte[]'"/>
+		            <field name="operator" colSpan="12" required="true" selection-in="['=','!=','isNull','notNull']" hidden="true" showIf="targetType == 'ManyToOne' || targetType == 'OneToOne'"/>
+		            <field name="operator" colSpan="12" required="true" selection-in="['in','notIn','like','notLike','isNull','notNull','include','notInclude']" hidden="true" showIf="targetType == 'ManyToMany'"/>
```

Comment:
Keep 'in', 'notIn' or 'include','notInclude' but noth, check which one is working. 
The 'like' and 'notLike' operators are for string .